descriptors/ModuleDescriptor-template.json

```diff
@@ -1881,6 +1903,17 @@
       ],
       "visible": false
     },
+    {
+      "permissionName": "modperms.circulation.loans.change-due-date.post",
+      "displayName": "module permissions for one op",
+      "description": "to reduce X-Okapi-Token size",
+      "subPermissions": [
+        "circulation-storage.loans.item.put",
```

Comment:
Could you please re-check if we did not miss permissions. As I see we require at least `item.get` and `loan.get`.

Reply:
It looks like I may need to add permissions for `scheduled-notice-storage`, such as some of those found under:

[descriptors/ModuleDescriptor-template.json#L629](https://github.com/folio-org/mod-circulation/blob/CIRC-647/descriptors/ModuleDescriptor-template.json#L629)